### PR TITLE
Update with some more resources

### PR DESCRIPTION
**Pull request by alexrodba**
Added some interesting blog posts I wrote and our fully-remote company.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,6 +100,11 @@ A curated list of awesome [remote working](https://en.wikipedia.org/wiki/Telecom
   1. [Things To Watch Out For While Working Remotely](https://www.toptal.com/remote/remote-work-burnout-a-cautionary-tale)
   1. [How to Work Remotely and Still Be the Best](https://www.toptal.com/freelance/how-to-work-remotely-and-still-be-the-best)
   1. [The Ultimate Remote Team Culture](https://www.toptal.com/remote/the-ultimate-remote-culture)
+  1. [How to Handle Client Work Remotely: Our Communication Stack](https://marsbased.com/blog/2015/12/07/how-to-handle-client-work-remotely-our-communication-stack/)
+  1. [Our Six Tips for Working Remotely](https://marsbased.com/blog/2015/06/15/six-tips-for-remote-working/)
```
Comment (RichardLitt) [inline README.md]: This is a good article - but I don't think this list is awesome. It's a listicle. Can you remove it?

File: README.md [hunk 2]
```diff
@@ -100,6 +100,11 @@ A curated list of awesome [remote working](https://en.wikipedia.org/wiki/Telecom
   1. [Things To Watch Out For While Working Remotely](https://www.toptal.com/remote/remote-work-burnout-a-cautionary-tale)
   1. [How to Work Remotely and Still Be the Best](https://www.toptal.com/freelance/how-to-work-remotely-and-still-be-the-best)
   1. [The Ultimate Remote Team Culture](https://www.toptal.com/remote/the-ultimate-remote-culture)
+  1. [How to Handle Client Work Remotely: Our Communication Stack](https://marsbased.com/blog/2015/12/07/how-to-handle-client-work-remotely-our-communication-stack/)
```
Comment (RichardLitt) [inline README.md]: This is good.

File: README.md [hunk 3]
```diff
@@ -100,6 +100,11 @@ A curated list of awesome [remote working](https://en.wikipedia.org/wiki/Telecom
   1. [Things To Watch Out For While Working Remotely](https://www.toptal.com/remote/remote-work-burnout-a-cautionary-tale)
   1. [How to Work Remotely and Still Be the Best](https://www.toptal.com/freelance/how-to-work-remotely-and-still-be-the-best)
   1. [The Ultimate Remote Team Culture](https://www.toptal.com/remote/the-ultimate-remote-culture)
+  1. [How to Handle Client Work Remotely: Our Communication Stack](https://marsbased.com/blog/2015/12/07/how-to-handle-client-work-remotely-our-communication-stack/)
+  1. [Our Six Tips for Working Remotely](https://marsbased.com/blog/2015/06/15/six-tips-for-remote-working/)
+  1. [Dealing With Your First Hire in a Distributed Team](https://marsbased.com/blog/2014/10/25/dealing-with-your-first-hire-in-a-distributed-team/)
```
Comment (RichardLitt) [inline README.md]: I'm not sure this is entirely relevant to remote working; it covers only a particular thing, and there's no follow up later. I'd rather this wasn't in this PR.

File: README.md [hunk 4]
```diff
@@ -100,6 +100,11 @@ A curated list of awesome [remote working](https://en.wikipedia.org/wiki/Telecom
   1. [Things To Watch Out For While Working Remotely](https://www.toptal.com/remote/remote-work-burnout-a-cautionary-tale)
   1. [How to Work Remotely and Still Be the Best](https://www.toptal.com/freelance/how-to-work-remotely-and-still-be-the-best)
   1. [The Ultimate Remote Team Culture](https://www.toptal.com/remote/the-ultimate-remote-culture)
+  1. [How to Handle Client Work Remotely: Our Communication Stack](https://marsbased.com/blog/2015/12/07/how-to-handle-client-work-remotely-our-communication-stack/)
+  1. [Our Six Tips for Working Remotely](https://marsbased.com/blog/2015/06/15/six-tips-for-remote-working/)
+  1. [Dealing With Your First Hire in a Distributed Team](https://marsbased.com/blog/2014/10/25/dealing-with-your-first-hire-in-a-distributed-team/)
+  1. [How a Week of Remote Work Boosts my Productivity Enormously](http://www.alexrodba.com/2016/07/24/How-a-week-of-remote-work-boosts-my-productivity-enormously.html)
```
Comment (RichardLitt) [inline README.md]: 404?

File: README.md [hunk 5]
```diff
@@ -100,6 +100,11 @@ A curated list of awesome [remote working](https://en.wikipedia.org/wiki/Telecom
   1. [Things To Watch Out For While Working Remotely](https://www.toptal.com/remote/remote-work-burnout-a-cautionary-tale)
   1. [How to Work Remotely and Still Be the Best](https://www.toptal.com/freelance/how-to-work-remotely-and-still-be-the-best)
   1. [The Ultimate Remote Team Culture](https://www.toptal.com/remote/the-ultimate-remote-culture)
+  1. [How to Handle Client Work Remotely: Our Communication Stack](https://marsbased.com/blog/2015/12/07/how-to-handle-client-work-remotely-our-communication-stack/)
+  1. [Our Six Tips for Working Remotely](https://marsbased.com/blog/2015/06/15/six-tips-for-remote-working/)
+  1. [Dealing With Your First Hire in a Distributed Team](https://marsbased.com/blog/2014/10/25/dealing-with-your-first-hire-in-a-distributed-team/)
+  1. [How a Week of Remote Work Boosts my Productivity Enormously](http://www.alexrodba.com/2016/07/24/How-a-week-of-remote-work-boosts-my-productivity-enormously.html)
+  1. [How to Keep Your Remote Workers Productive](http://www.alexrodba.com/2016/03/22/How-To-Keep-Your-Remote-Workers-Productive.html)
```
Comment (RichardLitt) [inline README.md]: Also 404.

File: README.md [hunk 6]
```diff
@@ -290,7 +296,7 @@ A curated list of awesome [remote working](https://en.wikipedia.org/wiki/Telecom
   1. [Prezly](https://www.prezly.com/) - SaaS PR platform. PHP, React, JS, Postgres. 100% remote.
   1. [ProofHQ, a Workfront company](https://www.proofhq.com/job-openings.html) - Online proofing solution. PHP, JavaScript, Java, .NET, MySQL, Redis
   1. [RainforestQA](https://www.rainforestqa.com/jobs/) - On demand QA as a service.
-  1. [Real HQ](https://realhq.com) - Maker of Agent Pronto, software designed to help make home buying and selling a better experience. We've been remote since the start, now 50+ spread across the globe. Ruby, Rails, React.js, iOS/Android.
+  1. [Real HQ](https://realhq.com) - Maker of Agent Pronto, software designed to help make home buying and selling a better experience. We'''ve been remote since the start, now 50+ spread across the globe. Ruby, Rails, React.js, iOS/Android.
```
Comment (RichardLitt) [inline README.md]: Can you fix this?

**Comment by RichardLitt** [comment]
ping @alexrodba.

**Comment by RichardLitt** [comment]
Closing this as it has not been resolved in a timely fashion. Feel free to ping me to reopen if you come back to this.

**Comment by alexrodba** [comment]
Hi @RichardLitt just seen this. Happy to fix it. Apologies: Unroll.me screwed up some of my email subscriptions :)

**Comment by RichardLitt** [comment]
Sounds good @alexrodba! No rush. Reopened.

File: README.md (hunk 2 above)
Comment (alexrodba) [reply README.md]: Thanks!

File: README.md (hunk 1 above)
Comment (alexrodba) [reply README.md]: Done!

File: README.md (hunk 3 above)
Comment (alexrodba) [reply README.md]: Done!

File: README.md (hunk 4 above)
Comment (alexrodba) [reply README.md]: Fixed thanks

File: README.md (hunk 5 above)
Comment (alexrodba) [reply README.md]: Fixed, thanks!

File: README.md (hunk 6 above)
Comment (alexrodba) [reply README.md]: Fixed!

**Comment by RichardLitt** [comment]
Merged! c24088158ade549838cdefe46e171568ce1a2348

Thank you!

**Comment by alexrodba** [comment]
No, thank you for the good job and for keeping this valuable! Let me know if I can contribute further in any way and I will happily do it.

**Pull request closed without merging**